test/end-to-end/offer.js

```diff
@@ -355,6 +355,28 @@ describe("Offer endpoint tests", () => {
                 expect(created_offer).toHaveProperty("ownerName", test_company.name);
                 expect(created_offer).toHaveProperty("ownerLogo", test_company.logo);
             });
+
+            test("Should succeed to create an offer if the description's length is only shorter than the max \
+            without HTML tags", async () => {
+
+                const offer_params = {
+                    ...generateTestOffer(),
+                    description: `<h1>${"a".repeat(OfferConstants.description.max_length)}</h1>`,
+                    owner: test_company._id,
+                };
+
+                const res = await request()
+                    .post("/offers/new")
+                    .send(withGodToken(offer_params));
+
+                expect(res.status).toBe(HTTPStatus.OK);
+
+                const created_offer_id = res.body._id;
+                const created_offer = await Offer.findById(created_offer_id);
+
+                expect(created_offer).toBeDefined();
+                expect(created_offer).toHaveProperty("description", offer_params.description);
+            });
```

Comment:
Add a test for no tags as well (where the description is only text) and another one with `description: <h1>${"a".repeat(OfferConstants.description.max_length)+1}</h1>` asserting it fails

Reply:
The tests without tags are already done [here](https://github.com/NIAEFEUP/nijobs-be/blob/e158b035e0efee618a351563620724ad05cc5064/test/utils/ValidatorTester.js#L236) so I'll just do the latter